src/navigation/orbitalnavigator.cpp

```diff
@@ -2084,6 +2085,17 @@ double OrbitalNavigator::rotationSpeedScaleFromCameraHeight(
         1.0;
 }
 
+// This should only be run on nodes and not on master
```

Comment:
Maybe add a `ghoul_assert` for this, instead of this comment?  

Or, put the comment in the header file instead, as part of a doxygen comment for the function. IIRC, we avoid comments over function in the cpp files

Reply:
Adding a ghoul_assert makes sense I think